contactaccessrights/js/src/access-rights/app.js

```diff
@@ -0,0 +1,30 @@
+define([
+  'common/angular',
+  'common/ui-select',
+  'common/directives/loading',
+  'access-rights/controllers/access-rights-ctrl',
+  'access-rights/controllers/access-rights-modal-ctrl',
+  'access-rights/models/region',
+  'access-rights/models/location',
+  'access-rights/models/right'
+], function (angular) {
+  angular.module('access-rights', [
+      'access-rights.controllers',
+      'access-rights.models',
+      'ui.select',
```

Comment:
Just nitpicking here, but for consistency's sake list the vendor/common modules first
```js
'ui.select',
'xeditable-civi',
'access-rights.controllers',
// etc
```